### ISSUE TITLE
Proposal to Rephrase Success Criterion 4.1.1

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#2676]
https://github.com/w3c/wcag/issues/2525 was closed, but not because the content of the ticket was resolved. Clarification of the issue is still necessary, regardless of whether the author of the ticket wants it or not.

I would also like to point out that the proposal in the https://github.com/w3c/wcag/issues/2649 ticket is closely related to this. Both tickets should be reopened or followed up here.

**Comment by yatil** [comment]
In the [Web Accessibility Slack](https://web-a11y.slack.com/archives/C7KTV5CBA/p1663940462547839), the claim is that 4.1.1 would narrowly only apply to SGML/XML parsing. For example, a  `<p>` nested inside of an `<h1>` would pass 4.1.1 despite HTML (living standard) says it is not allowed. Only nesting in the following way would not conform here: `<h1><p>…</h1></p>`

As this is irrelevant for the created DOM according to HTML (living standard) parsing rules, it feels useless to have this Success Criterion at all. It would be more useful if the SC would be adapted to mean “nest according to the rules of the markup language you actually use” instead of “nest according to the rules of a different markup language”.

[Here’s the whole argumentation, including a suggestion for re-wording the SC](https://cstrobbe.gitlab.io/A11yWorks/wcagtests/html5/sc_4.1.1_syntax/index.html).

I think it’s worth the discussion if this SC should be adapted to fit modern standards (for example with custom elements, almost any element name would be correct to use and there are little ways to know if a <divs> element is a typo or a custom element). Furthermore, the syntax of the unparsed HTML has so little significance anymore now that a lot of DOM manipulation happens in the browser, after parsing.

The interpretation of this SC that the goal is to have a valid content model as the result of the parsing of the syntax plus DOM manipulations is practical, useful, and actually leads to more accessible content – instead of chasing theoretical syntactical errors.

Considering that this is a contentious and mostly useless SC, I would lean on the side of either simplifying it (“Elements are used and nested according to the specification of the markup language used.”) or removing it in WCAG 2.2.

**Comment by JAWS-test** [comment]
closed because https://github.com/w3c/wcag/issues/2525 is open again

**Comment by bruce-usab** [comment]
> I think it’s worth the discussion if this SC should be adapted to fit modern standards....

I recall that was contemplated early on with the 2.2 work.  There was pretty quick consensus that AGWG should not change 2.0/2.1 SC.  A new SC, requiring validity or what not, was a possibility.

My recollecting is there was also conversations about backward compatibility meaning AGWC _could_ make 4.1.1 (or other SC) stricter in 2.2 (as compared to 2.1/2.0).  Personally, I was not really comfortable with that idea, so I am relieved that it didn't come up.

FWIW, Promoting SC 2.4.7 Focus Visible from Level AA to Level A is okay with me.

> Considering that this is a contentious and mostly useless SC, I would lean on the side of either simplifying it (“Elements are used and nested according to the specification of the markup language used.”) or removing it in WCAG 2.2.

FWIW, removing 4.1.1 was contemplated with 2.2.  The idea never did get much traction.
